diff --git a/docker_registry/run.py b/docker_registry/run.py
index fb9fe39..a286b1c 100644
--- a/docker_registry/run.py
+++ b/docker_registry/run.py
@@ -55,6 +55,7 @@ def run_gunicorn():
 
     args = [
         gunicorn_path, 'gunicorn',
+        '--preload',
         '--access-logfile', env.source('GUNICORN_ACCESS_LOG_FILE'),
         '--error-logfile', env.source('GUNICORN_ERROR_LOG_FILE'),
         '--max-requests', '100',
